api/proto/teleport/legacy/types/types.proto

```diff
@@ -4588,6 +4588,10 @@ message OIDCConnectorMFASettings {
   // Prompt is an optional OIDC prompt. An empty string omits prompt.
   // If not specified, it defaults to select_account for backwards compatibility.
   string prompt = 5;
+  // MaxAge is the amount of time in nanoseconds that an IdP session is valid for. Defaults to
```

Comment:
It seems to cause issues - https://github.com/gravitational/teleport/pull/29815#discussion_r1287449841

I'll give it a try though.

Reply:
```proto
google.protobuf.Duration max_age = 6 [(gogoproto.stdduration) = true];
```
Resulted in it not parsing from a string like `10s`.

```proto
google.protobuf.Duration max_age = 6;
```
Would not parse from an int or a string like `10s`, seems completely broken. Looks like we map `- Mgoogle/protobuf/duration.proto=github.com/gogo/protobuf/types`, and gogoproto ofc has issues so I'm guessing changing this would be a huge pain.

```proto
google.protobuf.Duration max_age = 6 [
    (gogoproto.stdduration) = true,
    (gogoproto.casttype) = "Duration"
  ];
```
This works, though I don't think it's any better than the int64 as it still relies on gogoproto to overrule the actual type. I think it's essentially the same, but we can go with this.

@codingllama any opinion on this?

Reply:
> codingllama any opinion on this?

Yes, gogo was a terrible idea.  :P


Why does it not parse? In what way does it break?

No strong opinions on my part about using `int64` here - I would prefer the stronger type but I know how much of a pain it can be on public-facing types (and on legacy/ to boot).